WrittenKitten.js

```diff
@@ -106,15 +104,15 @@ function show_kitten() {
 }
 
 function fetch_next_kitten() {
+	let flickr_search_term
 	if (getParameterByName("search")) {
 		// if they are using a URL param, take them very literally. They
 		// generally know what they're doing.
-		search_for = getParameterByName("search");
-		flickr_search_term = search_for + ',cute';
+		flickr_search_term = getParameterByName("search");
 	} else {
 		// add "cute" to search if item is selected from dropdown. it just
 		// works better that way.
-		flickr_search_term = search_for;
```

Comment:
This is the core of the PR, making the code do what the comments already said it should do :)